Vagrantfile

```diff
@@ -17,7 +17,7 @@ Vagrant.configure("2") do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     apt-get update
-    apt-get install -y nasm gccgo xorriso
+    apt-get install -y nasm gccgo xorriso make
```

Comment:
I thought `make` was already part of the xenial image. Did you encounter issues with `make` not being present?